README.md

```diff
@@ -46,6 +46,16 @@ groups or DNF options.
 
 [issue 8]: https://github.com/fedora-python/tox-github-action/issues/8
 
+## Changelog
+
+Until version v0.4, this action always used the latest [fedora-python-tox](https://hub.docker.com/repository/docker/fedorapython/fedora-python-tox)
```

Comment:
```suggestion
Until version 0.4, this action always used the latest [fedora-python-tox](https://hub.docker.com/repository/docker/fedorapython/fedora-python-tox)
```

This is probably too nit-picky to you, but I would consider not using the `v` prefix here, as it means "version". Alternatively:

```suggestion
Until v0.4, this action always used the latest [fedora-python-tox](https://hub.docker.com/repository/docker/fedorapython/fedora-python-tox)
```

Don't consider this important though.

Reply:
I like the first one.